KEYHOLDER.md

```diff
@@ -1,40 +1,84 @@
-# TUF Generation
+# Keyholder Responsibilities
 
-Pre-requisites:
-* A local Git installation and a Go development setup
-* SSH authentication for GitHub (see [here](https://docs.github.com/en/authentication/connecting-to-github-with-ssh))
-* Keyholders: A USB port connection for your hardware key (beware of using a remote connection; the keyholder should not assume that magic occurs during an SSH session)
+This document outlines the responsibilities of a root keyholder.
 
-0. **The keyholders and the conductor** should fork [this](https://github.com/sigstore/root-signing) git repository by clicking the "fork" button on GitHub. Then, set your `${GITHUB_USER}` with your GitHub username and execute the script:
+## Outline
 
-```
+Keyholders MUST subscribe to the [Sigstore Maintainer Calendar](https://calendar.google.com/calendar/u/0?cid=Y19ydjIxcDJuMzJsbmJoYW5uaXFwOXIzNTJtb0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) for upcoming root signing event. Root signing events are expected to occur about every 4-5 months. The next `v+1` version signing will be scheduled, and the `v+2` version will be tentatively scheduled. Testing will occur the week before the signing. Keyholders are required to communicate that they have completed the [pre-work](#signing-pre-work) to the orchestrator through [#sigstore-keyholder](https://sigstore.slack.com/archives/C03E4HP6RCK) Slack channel. All testing and signing events can occur asynchronously. Keyholders are expected to be "on-call" during the root signing window in case there is an issue.
+
+### Pre-requisites
+
+Ensure you have the following:
+- [ ] A local Git installation and a Go development setup
+- [ ] SSH authentication for GitHub (see [here](https://docs.github.com/en/authentication/connecting-to-github-with-ssh))
+- [ ] A USB port connection for your hardware key (beware of using a remote connection; the keyholder should not assume that magic occurs during an SSH session)
+
+### Signing pre-work
+
+During a root signing test, keyholders must complete the following steps in order:
+- [ ] Fork the [root-signing](https://github.com/sigstore/root-signing) repository by clicking the "fork" button on GitHub. 
+- [ ] Test binary build: Set your `${GITHUB_USER}` with your GitHub username and execute the script:
+```bash
 export GITHUB_USER=${YOUR_GITHUB_USER}
 ./scripts/step-0.sh
 ```
-
-The conductor should also set the online snapshot and timestamp keys, and the previous repository if it exists. The key references should have the a go-cloud style URI like `gcpkms://<some key>`.
-
+This will setup your fork and build the TUF binary to use for metadata generation.
+- [ ] (If you are a new keyholder) Test registering your new key: Do not use a key that you need -- this will wipe the key! Set the following environment variables, and then follow the steps in [Registering a new root key](#registering-a-new-root-key)
```

Comment:
Can we specify that you can use your real yubikey HSM key for this? I don't think we should expect they have a test key and the real key, unless we want to send them two.

Reply:
I read this as needing a dedicated key for root signing, not to use a key which is already in-use for a different purpose.

Reply:
fixed per Joshua's suggestion below.